### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/customize/color.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#791] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/customize/color.html

此页面指导你如何自定义主题颜色。 修改内置颜色 主题通过调色板控制颜色，你可能需要分为两种情况设置你的颜色： 此颜色在日间模式和夜间模式保持不变，如主题色。 此颜色在日间模式、夜间模式下不同，如背景色、字体、边框颜色等。 调色板文件是 VuePress 项目文件夹下的 .vuepress/styles/palette.scss 文件。 对于前者，你需要...

https://theme-hope.vuejs.press/zh/guide/customize/color.html

**Comment by zhanglinweigithub** [comment]
求助，请问如何在MD文件或vue文件中获得当前实用的主题色，换句话说就是读取scss文件中的变量值

**Reply by zhanglinweigithub** [reply]
我在使用VPCard组件，想设置背景色为当前主题色
